common/src/main/java/org/apache/rocketmq/common/attribute/AttributeParser.java

```diff
@@ -16,38 +16,44 @@
  */
 package org.apache.rocketmq.common.attribute;
 
-import com.google.common.base.Joiner;
 import com.google.common.base.Strings;
-
 import java.util.ArrayList;
 import java.util.HashMap;
 import java.util.List;
 import java.util.Map;
 
 public class AttributeParser {
+
+    public static final String COMMA = ",";
+
+    public static final String EQUAL_SIGN = "=";
+
+    public static final String PLUS_SIGN = "+";
+
+    private static final String MINUS_SIGN = "-";
```

Comment:
Pls use more descriptive names, like 'ATTR_ARRAY_SEPARATOR'.

Reply:
> Pls use more descriptive names, like 'ATTR_ARRAY_SEPARATOR'.

I will do, thanks for you suggest